### PR TITLE
Set linter

### PR DESCRIPTION
**Pull request by k1LoW**
I would like to implement Linter for healthy development.

Linters set up are my taste.

### CODE REVIEW

File: binary.go [hunk 1]
```diff
@@ -1,32 +1,37 @@
...
-type XfsQuotaBinary struct {
+type Binary struct {
 	// The path to xfs_quota binary
 	Path string
 }
 
-func (b *XfsQuotaBinary) Execute(ctx context.Context, stdout io.Writer, stderr io.Writer, args ...string) error {
-	cmd := exec.CommandContext(ctx, b.Path, args...)
+func (b *Binary) Execute(ctx context.Context, stdout io.Writer, stderr io.Writer, args ...string) error {
+	e, err := safeexec.LookPath(b.Path)
```
Comment (k1LoW) [inline binary.go:23]: Use safeexec

File: binary.go [hunk 2]
```diff
@@ -1,32 +1,37 @@
...
+	}
+	cmd := exec.CommandContext(ctx, e, args...)
 	cmd.Stdout = stdout
 	cmd.Stderr = stderr
 
 	return cmd.Run()
 }
 
-func (b *XfsQuotaBinary) Validate() error {
+func (b *Binary) Validate() error {
```
Comment (k1LoW) [inline binary.go:34]: Fix repitition

File: xfs_quota.go [hunk 3]
```diff
@@ -44,7 +43,7 @@ func NewClient(binaryPath string) (*XfsQuotaClient, error) {
 	return c, nil
 }
 
-func (c *XfsQuotaClient) GetBinaryVersion() (string, error) {
+func (c *Client) GetBinaryVersion() (string, error) {
```
Comment (k1LoW) [inline xfs_quota.go:46]: Fix repitition

File: Makefile [hunk 4]
```diff
@@ -2,6 +2,9 @@
 test:
 	go test ./... -coverprofile=coverage.out -covermode=count
 
+lint:
+	golangci-lint run ./...
```
Comment (ry023) [inline Makefile:6]: 👍 👍

**Review by ry023 — APPROVED** [review]
Appreciate it!!!!!

**Pull request merged**